### PR TITLE
refactor(core): a couple of minor changes to the early-event-contract

### PR DESCRIPTION
**Pull request by alan-agius4**
This changes include
- Using multi line comments to write JSDoc comments which improves DX
- Use `this.container` instead of `window.document.documentElement`

### CODE REVIEW

File: packages/core/primitives/event-dispatch/src/earlyeventcontract.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ export class EarlyEventContract {
           event.type,
           event,
           event.target as Element,
-          window.document.documentElement,
+          this.container,
```
Comment (AndrewKushnir) [inline packages/core/primitives/event-dispatch/src/earlyeventcontract.ts:55]: nit: can we refer to the `container` without `this` (to save some bytes)?
```suggestion
          container,
```

**Review by AndrewKushnir — APPROVED** [review]
👍

**Review by AndrewKushnir — APPROVED** [review]

**Review by iteriani — APPROVED** [review]

**Comment by alan-agius4** [comment]
[TGP](http://test/OCL:631401009:BASE:631662186:1715147785013:355b6dde)

**Comment by alan-agius4** [comment]
[TGP](http://test/OCL:631401009:BASE:631643683:1715180819562:3f332651)

**Comment by atscott** [comment]
This PR was merged into the repository by commit c0012865eba3717dc9d1d9d3a9d5b7df909897ff.

**Pull request closed without merging**